streams/src/main/java/org/apache/kafka/streams/state/WindowBytesStoreSupplier.java

```diff
@@ -33,11 +33,22 @@
      * It is also used to reduce the amount of data that is scanned when caching is enabled.
      *
      * @return number of segments
+     * @deprecated since 2.1. Use {@link WindowBytesStoreSupplier#segmentInterval()} instead.
      */
+    @Deprecated
     int segments();
 
     /**
-     * The size of the windows any store created from this supplier is creating.
+     * The size of the segments (in milliseconds) the store has.
+     * If your store is segmented then this should be the size of segments in the underlying store.
+     * It is also used to reduce the amount of data that is scanned when caching is enabled.
+     *
+     * @return size of the segments (in milliseconds)
+     */
+    long segmentInterval();
```

Comment:
nit: I just realized that in `SessionBytesStoreSupplier` we have `segmentIntervalMs`, maybe we should make them consistent? Such nit renaming does not need to recast the vote.

Reply:
Oops! Thanks for the catch.